### PR TITLE
v.what.strds.timestamp: fix v.what.rast runs twice

### PR DESCRIPTION
**Pull request by pesekon2**

### CODE REVIEW

File: grass7/vector/v.what.strds.timestamp/v.what.strds.timestamp.py [hunk 1]
```diff
@@ -123,12 +123,15 @@ def sample_absolute(input, layer, timestamp_column, column, t_raster,
         grass.run_command("g.region", n=treg['n'], s=treg['s'], e=treg['e'],
                           w=treg['w'], align=raster_map)
 
+        if i_flag is True:
+            flags = 'i'
+        else:
+            flags = ''
+
         # Sample spatio-temporally matching points and raster map
-        rast_what = Module('v.what.rast', map=input, layer=layer,
```
Comment (ninsbl) [inline grass7/vector/v.what.strds.timestamp/v.what.strds.timestamp.py:127]: Wouldn't it be enough to add `, run_=False` to the module object?

**Review by ninsbl — APPROVED** [review]
Good catch! Thanks. See one comment inline. But this implementation would be ok as well. Just a bit more code...

File: grass7/vector/v.what.strds.timestamp/v.what.strds.timestamp.py (hunk 1 above)
Comment (pesekon2) [reply grass7/vector/v.what.strds.timestamp/v.what.strds.timestamp.py:127]: True. As I have just never used `run_`, I don't know anything about speed, what is the recommended way, anything - you say that you would prefer to use the `run_` argument? It's not a problem to change the PR as it is going to be squashed in any case.

Comment (ninsbl) [reply grass7/vector/v.what.strds.timestamp/v.what.strds.timestamp.py:127]: I don`t have a strong opinion. Using `run_=False` saves us an `if-else` and personally I would probably prefer it. However, that is no big issue. So I have no objection against our solution...

**Pull request merged**